src/pages/BlogPage.jsx

```diff
@@ -183,7 +163,7 @@ export function BlogPostMarkdown(props) {
           <td
             style={{
               padding: 5,
-              fontFamily: "Roboto Serif",
+              fontFamily: "Merriweather",
```

Comment:
Intended?

Reply:
I don't remember the exact reason but there was some quirk about the roboto serif font that prevented the transition. If there's a hard brand rule we're bumping up against I can put roboto back in and see what the errors were.

Reply:
I don't remember the exact reason but there was some quirk about the roboto serif font that prevented the transition. If there's a hard brand rule we're bumping up against I can put roboto back in and see what the errors were.